devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/resteasy-reactive-qute-codestart/kotlin/src/main/kotlin/org/acme/SomePage.kt

```diff
@@ -0,0 +1,23 @@
+package org.acme
+
+import io.quarkus.qute.Template
+import io.quarkus.qute.TemplateInstance
+import javax.inject.Inject
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+import javax.ws.rs.Produces
+import javax.ws.rs.QueryParam
+import javax.ws.rs.core.MediaType
+
+@Path("/some-page")
+class SomePage {
+
+    @Inject
+    lateinit var page: Template
```

Comment:
It would probably best to use constructor injection

Reply:
@geoand any reason we don't do it in the guide? https://quarkus.io/guides/qute

Reply:
maybe you were talking about the `lateinit var`?

Reply:
We should be using constructor in general, but even more so in Kotlin since the field injection is ugly there.

Not doing in the guides is just because we are so used to field injection.

Reply:
ok fixed in java & kotlin for reactive and classic